docs/contribute/docs.md

```diff
@@ -0,0 +1,45 @@
+# Contribute to the documentation
+
+## Build the MyST guide documentation locally
+
+To build the MyST guide documentation:
+
+1. Clone this repository:
+
+   ```
+   git clone https://github.com/executablebooks/mystmd
+   ```
+2. [Install MyST Markdown by following these instructions](https://mystmd.org/guide/quickstart)
+3. Navigate to the docs folder:
+
+   ```
+   cd docs/
+   ```
+4. Start a MyST server to preview the documentation:
+
+   ```
+   myst start
+   ```
+
+This will build the documentation locally so that you can preview what your changes will look like.
+
+
+## How this relates to the documentation at mystmd.org
+
+The documentation in this repository serves the content that lives [at `mystmd.org/guide`](https://mystmd.org/guide).
+
+The full content and theme for [`mystmd.org`](https://mystmd.org) is [hosted at `curvenote/mystmd.org`](https://github.com/curvenote/mystmd.org), which pulls in the content from this folder along with several other documentation sources into one place.
+
+However, the documentation here can be built independently for previewing your changes.
+
+
+## The documentation theme infrastructure
+
+The MyST documentation uses a custom theme that inherits from a chain of base themes.
+Here's a brief overview of where to look for things:
+
+- The MyST documentation theme is a custom theme defined [at the `curvenote/mystmd.org` repository](https://github.com/curvenote/mystmd.org).
```

Comment:
We talked ages ago about moving this repository over, but I haven't got around to it. I think now is a good time. There is nothing curvenote specific in here other than the CDN that it is pulling the content from, I suggest we move that to EBP and generalize the CDN (e.g. load the content off of github).

Reply:
I agree with that - can we track this in a separate issue so that it doesn't block this PR? My goal was to just document the current state of things for a newcomer because it is kind of confusing to understand the lay of the land right now. (and I don't have the permissions, time, or knowledge to take action on your suggestion right now)

Reply:
I will transfer now, and we can document the CDN updates as an ongoing issue.